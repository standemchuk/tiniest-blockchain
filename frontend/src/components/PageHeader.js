import React from 'react'
import Grid from '@material-ui/core/Grid'
import Typography from '@material-ui/core/Typography'
import { withStyles } from '@material-ui/core/styles'

const styles = theme => ({
  header: {
    marginTop: 20,
    padding: theme.spacing.unit * 2
  }
})

export const PageHeader = (props) => (
  <Grid item xs={12}>
    <Typography
      className={props.classes.header}
      variant='display1'
      paragraph
      align={'center'}
    >
      {props.title}
    </Typography>
  </Grid>
)

export default withStyles(styles)(PageHeader)
