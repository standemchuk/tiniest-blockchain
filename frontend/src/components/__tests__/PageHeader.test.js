import React from 'react'
import { PageHeader } from '../PageHeader'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <PageHeader classes={{ header: 'header' }} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
