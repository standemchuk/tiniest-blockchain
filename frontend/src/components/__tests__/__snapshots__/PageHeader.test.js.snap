// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiGrid-item-2 MuiGrid-grid-xs-12-40"
>
  <p
    className="MuiTypography-root-97 MuiTypography-display1-101 MuiTypography-paragraph-115 MuiTypography-alignCenter-110 header"
  />
</div>
`;
